modules/oe_theme_helper/oe_theme_helper.module

```diff
@@ -32,7 +32,7 @@ function oe_theme_helper_page_attachments(array &$page) {
   // Load ECL component library assets for oe_theme and its subthemes.
   $active_theme = \Drupal::theme()->getActiveTheme();
   if ($active_theme->getName() === 'oe_theme' || array_key_exists('oe_theme', $active_theme->getBaseThemeExtensions())) {
-    $page['#attached']['library'][] = 'oe_theme/component_library_' . theme_get_setting('component_library', 'oe_theme');
+    $page['#attached']['library'][] = 'oe_theme/component_library_' . theme_get_setting('component_library');
```

Comment:
Can we default here to `ec` if no value is found? This might mitigate some situations when, after extending the oe_theme, sub theme config is not saved yet.